expected/wasm32-wasi/defined-symbols.txt

```diff
@@ -442,7 +442,6 @@ cimagl
 clearenv
 clearerr
 clearerr_unlocked
-clock
```

Comment:
Do we not want a superset of all symbols here?

Reply:
This diff happens because `clock` is moved to `libwasi-emulated-process-clocks.a` instead of `libc.a`.

I guess we could change `defined-symbols.txt` to include symbols from all the libraries, and not just `libc.a`. I'll look into that.

Reply:
Yup, that is what I was suggesting.  Nice!